org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/properties/SonarLintProjectPropertyPage.java

```diff
@@ -135,14 +135,14 @@ private void updateState() {
     Optional<EclipseProjectBinding> projectBinding = getProjectConfig().getProjectBinding();
     if (projectBinding.isPresent()) {
       boundDetails
-        .setText("Bound to the project '" + projectBinding.get().projectKey() + "' on server '" + serverName(projectBinding.get().serverId()) + "'");
+        .setText("Bound to the project '" + projectBinding.get().projectKey() + "' on server '" + serverName(projectBinding.get().connectionId()) + "'");
       bindLink.setText("<a>Update project binding</a>");
     } else {
       bindLink.setText("<a>Bind this Eclipse project to a SonarQube project</a>");
```

Comment:
Found a lonely SonarQube here, might need to be updated to SonarCloud/SonarQube?

Reply:
Fixed!